docs: add website setup to readme (#1395)
asyncapi/modelina: modelina-website/README.md

- `src/components/playground/PlaygroundOptions.tsx` is the main component that renders the options based on which output is selected.
- `src/components/playground/options` contain all the individual react components for showing the output options.
- `src/components/playground/Playground.tsx` is the main playground component, and is the one rendered by the playground page.
- `src/components/playground/GeneratedModels.tsx` is the playground component responsible for rendering the generated models.

> ## Run modelina locally

Reviewer: ```suggestion
## Run Modelina Website locally
```

Author: I have done this,

> Contributions to `modelina` check out [git-workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md)

Reviewer: ```suggestion
Checkout documentation that explains [git-workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md) used in our repositories. 
```

Author: I have resolved this.

1. Navigate to the website directory.

```bash

>     cd website

Reviewer: ```suggestion
    cd modelina-website
```

Author: Resolved this

>     cd website-website

Reviewer: @jonaslagoni @rukundob451 there is a still an error here 😓 
cd modelina-website

Author: I have checked this out now and have resolved it, thank you for the clarification.

```

2. Install all website dependencies. 

```bash
    npm install
```

3. Run the website locally.

```bash
    npm run dev
```

> 4. ccess the live development server at [localhost:3000](http://localhost:3000).

Reviewer: ```suggestion
4. Access the live development server at [localhost:3000](http://localhost:3000).
```

Author: Thanks for this clarification

- `src/components/playground/PlaygroundOptions.tsx` is the main component that renders the options based on which output is selected.
- `src/components/playground/options` contain all the individual react components for showing the output options.
- `src/components/playground/Playground.tsx` is the main playground component, and is the one rendered by the playground page.
- `src/components/playground/GeneratedModels.tsx` is the playground component responsible for rendering the generated models.

## Contributors flow

1. Fork the repository by clicking on `Fork` option on top right of the main repository.

2. Open Command Prompt on your local computer.

3. Clone the forked repository by adding your own GitHub username in place of `<username>`.
   For multiple contributions it is recommended to have [proper configuration of forked repo](https://github.com/asyncapi/community/blob/master/git-workflow.md).

```bash
    git clone https://github.com/<username>/website/

> ```

Reviewer 2: I think we should just link to https://github.com/asyncapi/community/blob/master/git-workflow.md and that's it, at least for the first 3 points 🙂 

No need to repeat what it say.

Author: Ohhhh, I get you @jonaslagoni. Let's do that indeed no need to repeat we shall just link to https://github.com/asyncapi/community/blob/master/git-workflow.md
